### DISCUSSION TITLE
System Requirements Specification (SRS) document

### DISCUSSION THREAD

**Discussion by TemirlanAidarov** [osmlab/name-suggestion-index#6221] [Q&A] [answered]
Hello!
I have just found about this project.
I am Computer Science student who takes a course related to Open Software Project.
We have an assignment to write/improve System Requirements Specification (SRS) document for any project. Basically, the SRS is a document that describes the nature of a project, software or application. In simple words, SRS document is a manual of a project provided it is prepared before you kick-start a project/application.

Can I write/improve SRS document for this project?

**Comment by bhousel** [comment, marked as answer]
Thanks for the offer!  This project doesn't really have system requirements, and wouldn't be a good candidate for your assignment.

The name suggestion index is really just a collection of json files that match commonly mapped features in OpenStreetMap to their preferred tags and wikidata identifiers.

**Reply by TemirlanAidarov** [reply]
Ok, I think then I will find another project
Thank you for the answer!
